### PR TITLE
resolve CData section too big on Export grid issue 24311

### PR DESCRIPTION
**Pull request by gaurangpadhiyar007**
### Fixed Issues 
1. magento/magento2#24311: "CData section too big" error while accessing configurable product in backend 

### Manual testing scenarios (*)

1. At export history grid number of files more (>  20000)

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=24314) <br/>All committers have signed the CLA.

**Comment by Stepa4man** [comment]
@magento run all tests

**Comment by gaurangpadhiyar007** [comment]
@magento area adminhtml

**Comment by gaurangpadhiyar007** [comment]
@magento-cicd2 @magento-admin run area adminhtml

@magento-cicd2 @magento-admin run all tests

**Comment by gaurangpadhiyar007** [comment]
@magento run area adminhtml
@magento run all tests

**Comment by rodrigowebjump** [comment]
Hi @gaurangpadhiyar007 Please, 
Thanks for your contribution.
I kindly ask you to add an appropriate description and tests scenarios to your PR.

Please, do not forget to review the failing tests.

Thanks
Thanks

**Review by gaurangpadhiyar007 — COMMENTED** [review]
Issue of merging run test solved

**Comment by gaurangpadhiyar007** [comment]
@rodrigowebjump
 I have made all changes as per given errors but as I solve them new un-neccessary errors are generating by Automated testing tool, Please test the commit on Magento instance.

File: app/code/Magento/ImportExport/Ui/DataProvider/ExportFileDataProvider.php [hunk 1]
```diff
@@ -51,11 +60,14 @@ public function __construct(
         \Magento\Framework\Api\FilterBuilder $filterBuilder,
         DriverInterface $file,
         Filesystem $filesystem,
+        File $fileIO,
```
Comment (sidolov) [inline app/code/Magento/ImportExport/Ui/DataProvider/ExportFileDataProvider.php:63]: Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

**Review by sidolov — CHANGES_REQUESTED** [review]

**Comment by sidolov** [comment]
@gaurangpadhiyar007  I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**